tasks/create-pyxis-image/create-pyxis-image.yaml

```diff
@@ -79,10 +93,16 @@ spec:
             exit 1
         fi
 
-        DATA_FILE="$(workspaces.data.path)/$(params.dataPath)"
-        if [ ! -f "${DATA_FILE}" ] ; then
-            echo "No data JSON was provided."
-            exit 1
+        if [ -n "$(params.commonTag)" ]; then
+          TAG="$(params.commonTag)"
+        else
+          # DATA_FILE is only needed for defaultTag which is only needed if commonTag is empty
+          DATA_FILE="$(workspaces.data.path)/$(params.dataPath)"
+          if [ ! -f "${DATA_FILE}" ] ; then
+              echo "No data JSON was provided."
+              exit 1
+          fi
+          TAG=$(jq -r '.images.defaultTag' "${DATA_FILE}")
```

Comment:
Do we need to specify a default here? latest?

Reply:
Here I just moved stuff around - I didn't change the way this works if `commonTag` is not set. So I'd prefer to keep it as it is for now for non-rh-push use cases. If we need a default for those use cases, it should be handled in a separate story/PR.

Reply:
ok. thanks